src/pydata_sphinx_theme/assets/styles/extensions/_sphinx_design.scss

```diff
@@ -0,0 +1,61 @@
+/**
+ * Special-cases for the sphinx-design lib
+ * Mainly to make it compatible with the multime themes supported by pydata-sphinx-theme
+ */
+
+/*******************************************************************************
+ * shadows
+ */
+.sd-shadow-xs,
+.sd-shadow-sm,
+.sd-shadow-md,
+.sd-shadow-lg {
+  box-shadow: 0 0.5rem 1rem var(--pst-color-shadow);
+}
+
+/*******************************************************************************
+ * cards
+ */
+
+.sd-card {
```

Comment:
I know that sphinx-design uses many of its own CSS variables:

https://sphinx-design.readthedocs.io/en/latest/css_variables.html

If possible, I think it would be better to control the behavior here by using `--pst` variables to set `--sd` variables, rather than coding new CSS rules. Does that make sense?

Reply:
design-style `.css` is loaded after pydata-sphinx-theme one so I won't be able to overwrite anything (that's also why I need to use `!important` rules in lots of places). I think I can't.

```html
  <!-- Loaded before other Sphinx assets -->
  <link href="_static/styles/theme.css?digest=fb52004a9691e5b75515" rel="stylesheet">
<link href="_static/styles/pydata-sphinx-theme.css?digest=fb52004a9691e5b75515" rel="stylesheet">

    
  <link rel="stylesheet"
    href="_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    <link rel="stylesheet" type="text/css" href="_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="_static/graphviz.css" />
    <link rel="stylesheet" type="text/css" href="_static/../_font/fontawesome/css/all.min.css" />
    <link rel="stylesheet" type="text/css" href="_static/../_font/fontawesome/css/all.min.css" />
    <link rel="stylesheet" type="text/css" href="_static/design-style.b7bb847fb20b106c3d81b95245e65545.min.css" />
```

Reply:
When two selectors have the same "specificity", then the latter definition is used. That's the situation here. You can work around this without `!important`s here by having a more specific selector -- like `body .sd-card` (that increases the specificity by 1, which might just be enough). :)

Most of the conflicting styles here aren't `!important`'d, so it's not a big deal. The issues I had with `!important` in that library are around this file: https://github.com/executablebooks/sphinx-design/blob/cca2cfb1b4c0a55ecf6661889c52ea320d42f58f/style/_display.scss

Furo has a separate furo-extensions.css file that is loaded after the extensions, to deal with some of this -- but realistically, unless you're overriding `!important`s (you're not, AFAICT), you can typically get away with a more specific selector. :)

Reply:
Ooh that is a great point. Can we try increasing the specificity just a bit to see if we can add these fixes without the "important" lines?